### PR TITLE
Module3 homework completed.

### PR DESCRIPTION
**Pull request by markrenaud**
Part 1, 2 & 3 of Module 3 homework completed ready for review.

### CODE REVIEW

File: Module-3/Homework.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+ `let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"`
+ 
+ Iterate over sentence counting the vowels (a, e, i, o, u), ignoring the case.
+ */
+let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"
+let vowels: [Character] = ["a", "e", "i", "o", "u"]
+var count = 0
+for character in sentence.lowercased() {
+    if vowels.contains(character) { count += 1 }
+}
```
Comment (niknovak) [inline Module-3/Homework.playground/Contents.swift:23]: Perfectly fine and readable, but for the reference, we can make this even shorter. For example, by using `filter` method on an array and then counting the elements.

File: Module-3/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+#### _f) List three data types you have used in Swift._
+
+- `String`
+- `Bool`
+- `Int`
+
+#### _g) To execute alternative code when the condition of an if statement is not met, you can use what clause?_
+
+`else` clause
```
Comment (niknovak) [inline Module-3/README.md:69]: Let's also not forget about the `else if` statement. ;)

**Review by niknovak — APPROVED** [review]
Perfect 👌

File: Module-3/Homework.playground/Contents.swift (hunk 1 above)
Comment (markrenaud) [reply Module-3/Homework.playground/Contents.swift:23]: Yes, filter would make this much cleaner.

File: Module-3/README.md (hunk 2 above)
Comment (markrenaud) [reply Module-3/README.md:69]: Noted.  I would consider an `else if` to still be an `if` statement.  My logic thus read the question as `any if` (`if` or `else if`) ...

**Pull request merged**